aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/exceptions/HiddenStackTraceException.java

```diff
@@ -0,0 +1,24 @@
+package com.castsoftware.aip.console.tools.core.exceptions;
+
+public class HiddenStackTraceException extends Exception {
```

Comment:
Leon, u misunderstand the requirement

Reply:
Wasn't  to just remove the stack trace?

Reply:
Now stack trace removed and the default message from API displayed according to our discussion

Reply:
Clearly, when
- verbose ON : stack trace can be output (display)
- verbose OFF: no stack trace